rcl/test/rcl/test_validate_enclave_name.cpp

```diff
@@ -82,4 +85,6 @@ TEST(TestValidateEnclaveName, test_validation_string) {
       "Enclave '" << enclave << "' failed with '" << validation_result << "'.";
     EXPECT_NE(nullptr, rcl_enclave_name_validation_result_string(validation_result)) << enclave;
   }
+  EXPECT_NE(nullptr, rcl_enclave_name_validation_result_string(-1));  // invalid result
```

Comment:
What's the expected error string? Can you EXPECT_STREQ to that?

Reply:
I could. But since the rest of the test doesn't and it's of no consequence to behavior, it's not clear it adds any value.

Reply:
You're expecting an error. The value is that you're testing that the correct error is caught.

Reply:
I can't say I agree with re-stating an implementation defined constant in the test, but I don't feel that strongly about it either. See https://github.com/ros2/rcl/pull/682/commits/38103ef9d632b2bc6f7d75cc997d39b3c513b2d5.

Reply:
Is it possible that these tests are linked against a different implementation? If so, then I would agree with you.

Reply:
Not really, constants are hard-coded in `rcl`.